pkg/apis/openebs.io/lvm/v1alpha1/lvmvolume.go

```diff
@@ -62,9 +62,14 @@ type VolumeInfo struct {
 
 	// VolGroup specifies the name of the volume group where the volume has been created.
 	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:MinLength=1
```

Comment:
remove the required option and add omitempty, `json:"volGroup,omitempty"`.

Reply:
I see `VolumeInfo` struct is also being used in LVMSnapShot crd. I was not sure if removing the required validation has any effect on it. Can you help me to understand what required validation does for string type? Does api-server check for string being non-empty in case of required? I'm not able to find any official documentation about the same.